specviz/widgets/arithmetic_editor.py

```diff
@@ -0,0 +1,230 @@
+import astropy.units as u
```

Comment:
This looks like a duplicate file?

Reply:
Yeah, this one needs to go. The arithmetic editor needs to live in the `plugin` directory.